Development/nmos/node_resources.cpp

```diff
@@ -125,6 +126,26 @@ namespace nmos
             }
         }
 
+        if (0 <= nmos::fields::control_protocol_ws_port(settings))
+        {
+            for (const auto& version : nmos::is12_versions::from_settings(settings))
+            {
+                auto ncp_uri = web::uri_builder()
+                    .set_scheme(nmos::ws_scheme(settings))
+                    .set_port(nmos::fields::control_protocol_ws_port(settings))
+                    .set_path(U("/x-nmos/ncp/") + make_api_version(version));
+                auto type = U("urn:x-nmos:control:ncp/") + make_api_version(version);
```

Comment:
make_api_version called twice. Maybe saving it in one call is better?

Reply:
Yes, but this is using the same style as other uri builds within the make_device function. This call is only being executed once in the node initialisation.